src/quote.ts

```diff
@@ -99,7 +100,7 @@ export const quoteResponseSchema = z.object(
       fiatAmount: z.string(),
       cryptoAmount: z.string(),
       guaranteedUntil: z.string(),
-      quoteId: z.string(),
+      quoteId: z.string().optional(),
```

Comment:
if we make this optional, should we add extra validation when we get quotes to validate a quoteId is present?